content/rs/concepts/high-availability/clustering.md

```diff
@@ -31,18 +31,15 @@ In clustering, these terms are commonly used:
 - Slot or Hash Slot - The result of the hash calculation.
 - Shard - Redis process that is part of the Redis clustered database.
 
-## When to use sharding
+## When to use clustering
 
-Sharding the keyspace is an efficient way of scaling Redis that should
-be employed when:
+Clustering is an efficient way of scaling Redis that should be used when:
 
-- The dataset is large enough to benefit from using the RAM resources
-    of more than one server. We recommend sharding a dataset
-    once it reaches a size of 25 GB (50 GB for RoF).
-- The operations performed against the database are CPU-intensive,
-    resulting in performance degradation. By having multiple CPU cores
-    manage the database's shards, the load of operations is distributed
-    among them.
+- The dataset is large enough to benefit from using the RAM resources of more than one node.
```

Comment:
Using the phrase - "enable clustering" is a little bit confusing because the user has no option to disable or enable the clustering in the cloud.

So maybe it can be rephrased to - 
"We recommend using the Redis Cluster technology to create multiple shards of...."

Reply:
@orshwartzredislabs That's RS docs, so better be explicit about the steps. We can assume we are talking to customers using RS and not cloud users